### PR TITLE
feat: add AX3600 support (not completed yet)

### PR DESCRIPTION
**Pull request by fredliang44**
plugin needs to be migrate to `linux_aarch`:

- [x] ShadowSocks
- [ ] KoolProxy
- [ ] Aria2
- [ ] VsFtp
- [ ] kms
- [ ] Frpc
- [ ] Ngrok
- [x] WebShell
- [ ] TinyProxy
- [ ] Entware
- [ ] KodExplorer
- [ ] EasyExplorer
- [ ] HttpFile
- [ ] VerySync
- [ ] FastDick
- [ ] FireWall
- [ ] JetBrains
- [ ] QianDao
- [ ] FileBrowser
- [ ] ZeroTier
- [ ] MIWIFI
- [ ] AliDDNS
- [ ] BaiduPCS
- [ ] DropBear
- [ ] Frps
- [ ] PPTPD
- [ ] SmartDNS
- [ ] SSServer
- [ ] Transmission
- [ ] WebD

### CODE REVIEW

**Comment by fredliang44** [comment]
dnsmasq inside AX3600 seems broken
```shell
root@XiaoQiang:~# /etc/init.d/dnsmasq start
Command failed: Not found
Command failed: Invalid argument
```

File: apps/shadowsocks/scripts/shadowsocks.sh [hunk 1]
```diff
@@ -325,6 +325,7 @@ ipset_rules_smartdns() {
     cp -rf ${mbroot}/apps/${appname}/config/gfwlist.conf ${mbtmp}/gfwlist.conf
     sed -i 's/7913/15353/g' ${mbtmp}/gfwlist.conf
     sed -i '/^server/d' ${mbtmp}/gfwlist.conf
+    mkdir -p ${mbtmp}
```
Comment (monlor) [inline apps/shadowsocks/scripts/shadowsocks.sh:328]: 这个命令在base里面已经加了

File: apps/fastdick/scripts/fastdick.sh [hunk 2]
```diff
@@ -30,7 +30,7 @@ start () {
     logsh "【$service】" "正在启动${appname}服务... "
     # open_port
     # write_firewall_start
-    nohup ${mbroot}/apps/${appname}/bin/${appname} &> ${mbroot}/var/log/${appname}.log & 
+    ${mbroot}/apps/${appname}/bin/${appname} &> ${mbroot}/var/log/${appname}.log &
```
Comment (monlor) [inline apps/fastdick/scripts/fastdick.sh:33]: 这里直接改成`daemon ${mbroot}/apps/${appname}/bin/${appname}`吧，方便以后统一修改

File: apps/mixbox/bin/base [hunk 3]
```diff
@@ -194,7 +194,7 @@ remove_firewall_start(){
 
 daemon() {
 	local binname="$(basename "${1}")"
-	nohup $@ &> ${mbroot}/var/log/${binname}.log &
+	$@ &> ${mbroot}/var/log/${binname}.log &
```
Comment (monlor) [inline apps/mixbox/bin/base:197]: 要不判断下nohup命令是否存在？存在则使用nohup

File: apps/shadowsocks/scripts/shadowsocks.sh (hunk 1 above)
Comment (fredliang44) [reply apps/shadowsocks/scripts/shadowsocks.sh]: 应该是需要mkdir `/tmp/etc/dnsmasq.d/` 已经重新提交了

File: apps/shadowsocks/scripts/shadowsocks.sh [hunk 4]
```diff
@@ -401,6 +401,7 @@ ipset_rules() {
     echo "ipset=/.microsoft.com/customize_white" >> ${mbtmp}/wblist.conf
   fi
   #黑白名单规则
+  mkdir /tmp/etc/dnsmasq.d/
```
Comment (monlor) [inline apps/shadowsocks/scripts/shadowsocks.sh:404]: 这个需要做一下判断，目录不存在则创建：`test ! -d /tmp/etc/dnsmasq.d && mkdir -p /tmp/etc/dnsmasq.d`

**Pull request merged**